src/app/events/page.tsx

```diff
@@ -56,7 +54,8 @@ type Props = {
 }
 
 const events = getEventsData()
-const { categoryOptions, validCategoryIds } = getEventsCategorySettings()
+const { options: categoryOptions, validIds: validCategoryIds } =
+  getFieldSettings({ collectionName: 'event_entries', fieldName: 'category' })
```

Comment:
Same question here. Couldn't we do this?

```ts
const { options: categoryOptions, validIds: validCategoryIds } =
  getCategorySettings({ collectionName: 'event_entries' })
```

Reply:
@CharlyMartin - I'm guessing because of `fieldName` - But we could rework `getCategorySettings` to accept `fieldName` ..

Reply:
I see!

I agree that `getCategorySettings` is likely obsolete now. We can use `getFieldSettings` instead; I find this more intuitive.

Regarding the renaming, I would go with `getCMSFieldOptions` and have that function return only an array of options.

We can create a simple util function to extract ids from the list of options when we need it, such as:

```ts
function getIdsFromOptions<O extends OptionType>(options: Array<O>) {
  return options.map((option) => option.id)
}

const validIds = getIdsFromOptions(options)
```

What do you think?